spyder/preferences/languageserver.py

```diff
@@ -608,30 +607,356 @@ def mouseDoubleClickEvent(self, event):
 class LSPManagerConfigPage(GeneralConfigPage):
     """Language Server Protocol manager preferences."""
     CONF_SECTION = 'lsp-server'
-    NAME = _('Language Server Protocol')
+    NAME = _('Completion and linting')
     ICON = ima.icon('lspserver')
 
     def setup_page(self):
+        newcb = self.create_checkbox
+
+        # --- Code completion ---
+        # Introspection group
+        introspection_group = QGroupBox(_("Introspection"))
+        completion_box = newcb(_("Enable code completion"), 'code_completion')
+        goto_definition_box = newcb(
+            _("Go to definitions"),
+            'jedi_definition',
+            tip=_("If this option is enabled, doing a left mouse click on\n"
+                  "an object name while pressing the Ctrl key will go to\n"
+                  "that object's definition (if resolved)."))
+        follow_imports_box = newcb(_("Follow imports when going to a "
+                                     "definition"),
```

Comment:
I was referring to the same thing as the above, sorry, which you agreed it was a suboption. But its a moot point since like you said we don't have the UI yet.